File: src/targets/swift/urlsession/fixtures/text-plain.swift
import Foundation

let postData = Data("Hello World".utf8)

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.timeoutInterval = 10
request.allHTTPHeaderFields = ["content-type": "text/plain"]
request.httpBody = postData

do {
  let (data, _) = try await URLSession.shared.data(for: request)
  print(String(decoding: data, as: UTF8.self))
} catch {
  print(error, error.localizedDescription)
}

File: src/targets/javascript/jquery/fixtures/text-plain.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/anything',
  method: 'POST',
  headers: {
    'content-type': 'text/plain'
  },
  data: 'Hello World'
};

$.ajax(settings).done(res => {
  console.log(res);
});